test/TestUtilities/Test.Utility/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("ApiDesign", "RS0030:Do not used banned APIs", Justification = "This is a test helper utility.", Scope = "member", Target = "~M:Test.Utility.DebuggerUtils.WaitForDebugger")]
```

Comment:
Isn't this one already excluded by above?

```
  <ItemGroup>
    <AdditionalFiles Include="$(MSBuildThisFileDirectory)\BannedSymbols.txt" Condition="'$(TestProject)' != 'true'" />
  </ItemGroup>
```

Reply:
Test.Utility is actually compiled as a product project, even if it doesn't ship, because it builds on top of production code.